services/auth.js

```diff
@@ -0,0 +1,27 @@
+var passport                = require('passport');
+var FacebookStrategy        = require('passport-facebook');
+
+var log                     = require('../log');
+
+
+// Check that required keys are set
+
+const facebookCfg = {
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).